services/weblate/weblate-entity-count.service.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+const Joi = require('joi')
+const camelcase = require('camelcase')
+const { BaseJsonService } = require('..')
+const { nonNegativeInteger, optionalUrl } = require('../validators')
+const { metric } = require('../text-formatters')
+
+const schema = Joi.object({
+  count: nonNegativeInteger,
+}).required()
+
+const queryParamSchema = Joi.object({
+  server: optionalUrl.required(),
+}).required()
+
+class WeblateEntityCountBase extends BaseJsonService {
+  static category = 'other'
+
+  static buildRoute(entityName) {
+    return {
+      base: 'weblate',
+      pattern: entityName,
+      queryParamSchema,
+    }
+  }
+
+  static defaultBadgeData = { label: 'weblate', color: 'brightgreen' }
```

Comment:
Our badge defaults need to adhere to our [specification](https://github.com/badges/shields/blob/master/spec/SPECIFICATION.md) and also be rendered similar to how we handle other badges.

Here, I think that means we should use the entity name for the label, and we should also use the `informational` (blue) color as brightgreen is used to connote various forms of "success" or "passing"